src/openvassd.c

```diff
@@ -138,17 +136,10 @@ static openvassd_option openvassd_defaults[] = {
   {"auto_enable_dependencies", "yes"},
   {"nasl_no_signature_check", "yes"},
```

Comment:
This could be a good chance to move this to the prefs.c as well so that the -X in openvas-nasl (https://github.com/greenbone/openvas-scanner/blob/v6.0%2Bbeta1/nasl/nasl.c#L207-L208) isn't required by default anymore. Not sure if this could make sense in a separate PR.

Rationale:
The most common use-case for openvas-nasl is the development of new VTs where no signature are available at all.

Reply:
@cfi-gb Agree. Updated the PRs accordingly.